src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeEditor.cs

```diff
@@ -61,7 +63,15 @@ public override async Task<Solution> GetModifiedSolutionAsync()
                 //    these imports only were needed for the moved type, and so they shouldn't stay in the original
                 //    file.
 
-                var documentWithMovedType = await AddNewDocumentWithSingleTypeDeclarationAsync(newDocumentId).ConfigureAwait(false);
+                var unformattedDocumentWitMovedType = await AddNewDocumentWithSingleTypeDeclarationAsync(newDocumentId).ConfigureAwait(false);
+
+                var originalOptions = await unformattedDocumentWitMovedType.GetOptionsAsync().ConfigureAwait(false);
+
+                // When formatting a new document with a moved type, we don't need to modify the accessibility modifiers. They were already
+                // established for the existing type prior to being moved
+                var newDocumentOptions = (DocumentOptionSet)originalOptions.WithChangedOption(new Options.OptionKey(CodeStyleOptions2.RequireAccessibilityModifiers, unformattedDocumentWitMovedType.Project.Language), AccessibilityModifiersRequired.Never);
+                var newDocumentFormattingService = unformattedDocumentWitMovedType.GetRequiredLanguageService<INewDocumentFormattingService>();
+                var documentWithMovedType = await newDocumentFormattingService.FormatNewDocumentAsync(unformattedDocumentWitMovedType, newDocumentOptions, SemanticDocument.Document, default).ConfigureAwait(false);
```

Comment:
For some scenarios, @sharwell was suggesting that codefixes should never do a complete formatting. For example: https://github.com/dotnet/roslyn/issues/58078

Reply:
Unsure. I think whatever we decide should be same for "extract base" and "extract static" as well, since those are similar features that take existing code and move to a new document. 

I believe this is the right approach, but happy to hear Sam's thoughts on the matter.

Reply:
For this feature, it's fine to apply formatting as long as there are _absolutely no changes_ (not even whitespace changes) from the first line of the type that moved to its closing `}`.

Reply:
> (not even whitespace changes)

This could get tricky with namespace block changes for indentation.